packages/ListBoxBrowser/src/components/OptionsSelected/OptionsSelected.styles.js

```diff
@@ -0,0 +1,94 @@
+import { css } from "styled-components";
+import tokens from "@paprika/tokens/lib/tokens";
+import { spacer } from "@paprika/stylers/lib/helpers";
+
+export const container = css`
+  display: flex;
+  flex-wrap: wrap;
+  padding: ${tokens.space};
+`;
+
+export const optionStyles = css`
+  align-items: center;
+  background: ${tokens.backgroundColor.white};
+  border: 1px solid ${tokens.border.color};
+  border-radius: ${tokens.border.radius};
+  box-sizing: border-box;
+  color: ${tokens.color.black};
+  display: flex;
+  flex-basis: 100%;
+  flex-grow: 1;
+  font-size: 14px;
+  justify-content: flex-start;
+  margin: ${tokens.spaceSm};
+`;
+
+export const title = css`
+  background: ${tokens.backgroundColor.level0};
+  border-bottom: 1px solid ${tokens.border.color};
+  border-top: 1px solid ${tokens.border.color};
+  box-sizing: border-box;
+  padding: ${spacer(1)} ${spacer(1)} ${spacer(1)} 12px;
+  width: 100%;
+`;
+
+export const button = css`
+  border-top-left-radius: ${tokens.border.radius};
+  border-top-left-radius: ${tokens.border.radius};
+  flex-basis: 100%;
+  flex-grow: 1;
+  padding: ${tokens.spaceSm} ${tokens.space};
+`;
+
+export const counter = css`
+  background: ${tokens.color.white};
+  border: 1px solid ${tokens.border.color};
+  border-radius: ${tokens.border.radius};
+  font-size: 13px;
```

Comment:
use stylers helper

Reply:
yeesss

Reply:
what is the stylers/token for 13px?

Reply:
`stylers.fontSize(-2)` i believe.

Reply:
mmm I think at that level is better to have it as 13px is clearer